utils/_context/_scenarios.py

```diff
@@ -1021,6 +1009,8 @@ def configure(self, config):
                         "-f",
                         f"{tracer_version_dockerfile}",
                         "--quiet",
+                        "--platform",
```

Comment:
is this universal ? (working everywhere?)

Reply:
removed! Some leftovers from my local executions.....